src/cljs/athens/electron/fs.cljs

```diff
@@ -0,0 +1,169 @@
+                    [:transact athens-datoms/datoms]
+                    [:db-picker/add-new-db db-filepath]]})))
+
+(declare write-bkp)
+
+(defn sync-db-from-fs
+  "If modified time is newer, update app-db with m-time. Prevents sync happening after db is written from the app."
+  [filepath _filename]
+  (let [prev-mtime @(rf/subscribe [:db/mtime])
+        curr-mtime (.-mtime (.statSync fs filepath))
+        newer?     (< prev-mtime curr-mtime)]
+    (when newer?
+      (let [block-text js/document.activeElement.value
+            _          (.. js/navigator -clipboard (writeText block-text))
+            _          (write-bkp)
+            confirm    (js/window.confirm (str "New file found. Copying your current block to the clipboard, and saving your current db."
+                                               "\n\n"
+                                               "Accept changes?"))]
+        (when confirm
+          (rf/dispatch [:db/update-mtime curr-mtime])
+          (let [read-db (.readFileSync fs filepath)
+                db      (dt/read-transit-str read-db)]
+            (rf/dispatch [:reset-conn db])))))))
+
+
+(def debounce-sync-db-from-fs
+  (debounce sync-db-from-fs 1000))
+
+
+;; Watches directory that db is located in. If db file is updated, sync-db-from-fs.
+;; Watching db file directly doesn't always work, so watch directory and regex match.
+;; Debounce because files can be changed multiple times per save.
+(rf/reg-event-fx
+  :fs/watch
+  (fn [_ [_ filepath]]
+    (let [dirpath (.dirname path filepath)]
+      (.. fs (watch dirpath (fn [_event filename]
+                              ;; when filename matches last part of filepath
+                              ;; e.g. "first-db.transit" matches "home/u/Documents/athens/first-db.transit"
+                              (when (re-find #"conflict" (or filename ""))
+                                (throw "Conflict file created by Dropbox"))
+                              (when (re-find (re-pattern (str "\\b" filename "$")) filepath)
+                                (debounce-sync-db-from-fs filepath filename))))))
+    {}))
+
+
+;; TODO: parameterize so that can extend images-dir and db-filepath from filepath
+(rf/reg-event-fx
+  :fs/create-and-watch
+  (fn [{:keys [db]} [_ db-filepath]]
+    (let [default-dir (utils/default-dir)
+          images-dir (utils/default-image-dir-path)
+          db-filepath (utils/default-db-dir-path)]
+      (prn "CREATE"))))
+      ;(utils/create-dir-if-needed! default-dir)
+      ;(utils/create-dir-if-needed! images-dir)
+      ;{:db                (assoc db :db/filepath db-filepath)
+      ; :local-storage/set [:db/filepath db-filepath]
+      ; :fs/write!         [db-filepath (dt/write-transit-str (d/empty-db db/schema))]
+      ; :dispatch-n        [#_[:db/update-filepath db-filepath]
+      ;                     [:reset-conn athens-datoms/datoms]
+      ;                     [:db-picker/add-new-db db-filepath]]})))
```

Comment:
TODO: finish and parameterize so that can extend images-dir and db-filepath from filepath